server/v2/cometbft/abci.go

```diff
@@ -282,7 +285,7 @@ func (c *Consensus[T]) maybeRunGRPCQuery(ctx context.Context, req *abci.QueryReq
 	}
 
 	resp, err = queryResponse(res, req.Height)
-	return resp, isGRPC, err
+	return resp, true, err
```

Comment:
why r we assume isGRPC here is true ?

Reply:
I think we couldn't, its using as flag to run GRPC request or `app/p2p/store`. Can try by wrapping err with a special message.